explainer/index.html

```diff
@@ -0,0 +1,553 @@
+            meeting that definition. For example, a view could be considered a “screen” in a mobile app or a layer of
+            web content, such as a modal dialog.</p>
+        <p>Task flows are a series views that support a specified <a>user activity</a>. A task flow may include a subset of
+            items in a view or a group of views. Only the part of the views that support the user activity are included
+            in a test of the task flow.</p>
+
+        <p>Examples of a task flow include:</p>
+        <ul>
+            <li>Logging into a web site and being verified as a user;</li>
+            <li>Ordering an item, including the entire set of tasks from searching for the item, adding it to the
+                shopping cart, paying for it, and receiving confirmation;</li>
+            <li>Submitting tax information; and</li>
+            <li>Chatting with other users in a virtual reality environment.</li>
+        </ul>
+        <p>The product is the combination of all items, views, and task flows that comprise the web site, set of web
+            pages, web app, etc.</p>
+
+        <h4>Conditions</h4>
+        <p>Some tests only apply in certain situations. Testing may occasionally require determining and referencing
+            which specifications are being tested. Methods will note whether a test always applies or under what
+            conditions a test applies. Both quantitative and qualitative tests can be conditional.</p>
+        <p>Example conditions include:</p>
+        <ul>
+            <li>The language used may have different grammatical rules.</li>
+            <li>An interface with multiple contrast modes may have different contrast requirements than an interface
+                with only a default contrast mode.</li>
+        </ul>
+
+    <h4>Testing Assertions</h4>
+    <p>The quality of an assertion can be tested based on how well the assertion meets the documentation
+        requirements for assertions (See <a>Documenting Assertions</a>). Conforming to WCAG does not require testing
+        supporting documentation; however, organizations may decide to adopt additional documentation requirements
+        based on the procedure being asserted.</p>
+    </section>
+
+    <section id="conformance-models">
+        <h2>Draft Conformance Approach</h2>
+        <p>WCAG 3.0 will use a different conformance model than WCAG 2.0 in order to meet its requirements. Developing
+            and vetting the conformance model is a large portion of the work AG needs to complete over the next few
+            years. Drafts will include maturity models for public review and comment. </p>
+
+        <p>AG is exploring a model based on Foundational Requirements, Supplemental Requirements, and Assertions.</p>
+
+        <p>The most basic level of conformance will require meeting all of the Foundational Requirements. This set will be somewhat
+            comparable to WCAG 2.2 A & AA.</p>
```

Comment:
> This set will be somewhat comparable to WCAG 2.2 A & AA.

Suggest dropping "somewhat" qualifier.  I also suggest "WCAG 2.2 Level AA" or "all WCAG 2.2 Level A and Level AA success criteria".  I have been fighting the good fight that WCAG is more than the success criteria.

Reply:
(Chair hat off)
I'd like a little wiggle-room that's created by the "somewhat". When requirements are re-configured there will be all sorts of cases where things aren't quite the same as before. We're aiming for similar coverage and effort-level, but a little wiggle room will be useful so people don't say "but it doesn't include this things from SC 1.3.1" or similar.

Adjusting to "WCAG 2.2 level AA" seems reasonable.

Reply:
Discussed in [AG meeting on 2024-11-05](https://www.w3.org/2024/11/05-ag-minutes#41c5). Bruce accepted 'somewhat comparable'.